api/turing/db/database.go

```diff
@@ -0,0 +1,68 @@
+package db
```

Comment:
I'd suggest to name the package `database` and the DB instance as `db`

Reply:
done